### PR TITLE
Explain how to connect RIOT to Linux

### PR DESCRIPTION
**Pull request by x3ro**

### CODE REVIEW

**Comment by Kijewski** [comment]
In Debian you have to install netcat-openbsd instead of netcat-traditional. The latter does not know IPv6.

File: examples/ng_networking/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+To establish a connection between RIOT and the Linux host, the following dependencies are required:
+
+    sudo apt-get install bridge-utils openvpn
+
+After installing, create a tap interface (to which RIOT will connect) and a bridge (to which Linux will connect):
+
+    sudo brctl addbr br0
+    sudo ip link set br0 up
+    sudo openvpn --mktun --dev tap0
```
Comment (authmillenon) [inline examples/ng_networking/README.md]: Why not use `iproute2` here, too?

Comment (authmillenon) [inline examples/ng_networking/README.md]: (then you would not need install `openvpn`

File: examples/ng_networking/README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    sudo ip link set tap0 up
+    sudo brctl addif br0 tap0
```
Comment (authmillenon) [inline examples/ng_networking/README.md]: Here you can use `iproute2`, too. But why do you propose to create a bridge anyways?

Comment (x3ro) [inline examples/ng_networking/README.md]: I just took what worked for me in #2776, and also don't really know how I'd use iproute2 for this. Could you show me?

Comment (OlegHahm) [inline examples/ng_networking/README.md]: `sudo ip tuntap add dev <TAPNAME> user <USER> mode tap`
`sudo ip link set <TAPNAME> up`

File: examples/ng_networking/README.md (hunk 1 above)
Comment (authmillenon) [inline examples/ng_networking/README.md]:
```bash
sudo ip tuntap add tap0 mode tap user ${USER}
sudo ip link set tap0 up
```

As I said I don't think that the rest is needed, but if you want to create a bridge anyways:

```bash
sudo ip link add name br0 type bridge
sudo ip tuntap add tap0 mode tap user ${USER}
sudo ip link set dev tap0 master br0
sudo ip link set tap0 up
sudo ip link set br0 up
```

It's important to up the interface before the bridge. This way the bridge gets an IPv6 address related to the Hwaddr

**Comment by OlegHahm** [comment]
@x3ro , please move the branch to your fork.

**Comment by cgundogan** [comment]
corresponds to #3451

**Comment by x3ro** [comment]
@authmillenon What do you think?

File: examples/ng_networking/README.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
+# ng_networking example
+
+## Connecting RIOT native and the Linux host
+
+To establish a connection between RIOT and the Linux host, the following dependencies are required:
+
+* bridge-utils
```
Comment (authmillenon) [inline examples/ng_networking/README.md]: Isn't needed anymore.

**Comment by authmillenon** [comment]
Apart from the one issue: :+1:

**Comment by cgundogan** [comment]
ACK

**Comment by OlegHahm** [comment]
I think this documentation is in a good enough state to merge it after addressing @authmillenon's comment.

**Pull request merged**